pkg/helm/updates.go

```diff
@@ -159,8 +162,13 @@ func removeDuplicates(tags []string) []string {
 	return u
 }
 
-// TODO: Add caching
 func PullChartVersion(helmApp *apptypes.HelmApp, licenseID string, version string) (*bytes.Buffer, error) {
+	data, _ := getUpdateChartFromCache(helmApp, version)
```

Comment:
why ignore error?

Reply:
cache is not critical.  if it's not there, try to download the release

Reply:
might as well log the error then no?  seems like a trivial TODO